app/controllers/users_controller.rb

```diff
@@ -10,8 +10,10 @@ def create
 
     respond_to do |format|
       if @new_user.save
+        flash[:notice] = "Profile created successfully."
+        MsrMailer.welcome_email(@user).deliver_later
```

Comment:
@user doesn't exist here
you want to pass @new_user to welcome_email like so:
**MsrMailer.welcome_email(@new_user).deliver_later**